test/gtest/miopendriver_common.hpp

```diff
@@ -39,6 +39,20 @@
 using ::testing::HasSubstr;
 using ::testing::Not;
 
+#define CONV_FLOAT "conv"
```

Comment:
[Recommendation][style]
```suggestion
#define MIOPENDRIVER_BASEARG_CONV_FLOAT "conv"
```
here and below. Or something like
```C++
namespace miopendriver {
namespace basearg {
namespace conv {
const char float[] = "conv";
...
}}}
```

Reply:
Agreed, and updated.